Workspace/src/xbox_controller/src/teleop_node.cpp

```diff
@@ -42,18 +61,34 @@ class StateSelect {
   ros::Publisher drive_pub;
   ros::Publisher science_pub;
   ros::Publisher arm_pub;
-  enum topics { DRIVE, ARM, SCIENCE };
+  ros::Publisher autonomy_pub;
+  enum topics { DRIVE, ARM, SCIENCE, AUTONOMY };

```

Comment:
Add a NUM_TOPICS to the end of enum topics
`enum topics {DRIVE, ARM, SCIENCE, AUTONOMY, NUM_TOPICS}`
Then you can use that value of NUM_TOPICS instead of the `4` later on